backend/requirements/requirements-xenial.txt

```diff
@@ -11,7 +11,7 @@ pyasn1==0.1.9
 pyasn1_modules==0.0.7
 pycparser==2.14
 python_debian==0.1.28
-python_gnupg==0.3.8
+python_gnupg==0.4.1
```

Comment:
@NCommander: could you please clarify if this was really necessary and for which reason?

Actually this file is just needed in order to simulate what will be installed in the ubuntu xenial package that right now is shipping 0.3.8: https://packages.ubuntu.com/xenial/python-gnupg

The 0.4.1 will be included starting from ubuntu bionic https://packages.ubuntu.com/bionic/python-gnupg and will be included in the bionic packaging of globaleaks: https://github.com/globaleaks/GlobaLeaks/issues/2165

This change will require to be reverted

\cc @fpietrosanti